scripts/index.js

```diff
@@ -0,0 +1,108 @@
+// temperature changes number, color and landscape behavior
+const state = {
+    temperature : 73,
+    color : "orange",
```

Comment:
You should avoid class names like `orange`.  Classes should describe what they apply to, not the effect of adding the class.

(A class name like "sunny" or "cloudy" makes more sense for something like this.)